src/redux-ducks/extensions.js

```diff
@@ -102,6 +103,7 @@ export function installExtension(packageName: string) {
 
       dispatch({ type: INSTALL_SUCCESS, extension });
       dispatch({ type: RESET_CATALOGUE_STATE });
+      await fetchSources();
```

Comment:
I think the issue is that it needs to be wrapped in a `dispatch(...)`?

Reply:
This doesn't resolve the caching issue though

Reply:
We fetch all the sources again because when we install an extension there is a new sources and we don't know what those sources are. I manually remove sources on uninstall because we know what sources are associated with each installed extension so we know those sources will be gone once the extension is uninstalled.